### PR TITLE
Chore/ruby 2762 remove account email

### PR DESCRIPTION
**Pull request by PaulDoyle-DEFRA**
https://eaflood.atlassian.net/browse/RUBY-2762

### CODE REVIEW

File: features/bo/register_and_renew/view_details.feature [hunk 1]
```diff
@@ -3,7 +3,7 @@ Feature: NCCC agent views registrations from back office
   As an NCCC agent
   I want to view registration and renewal details on one service
   So that I can quickly answer user queries
-
+@mon4
```
Comment (timstone) [inline features/bo/register_and_renew/view_details.feature:6]: Debugging tag? could be removed

File: features/step_definitions/journey/registration_steps.rb [hunk 2]
```diff
@@ -124,14 +124,14 @@
...
 end
 
-Given("I create a new registration as {string}") do |account_email|
+Given("I create a new registration") do
   load_all_apps
   @tier = :upper
   @organisation_type = :limitedCompany
-  seed_data = SeedData.new("limitedCompany_complete_active_registration.json", "accountEmail" => account_email)
+  seed_data = SeedData.new("limitedCompany_complete_active_registration.json",
+                           "expires_on" => (DateTime.now + 30).to_s)
```
Comment (timstone) [inline features/step_definitions/journey/registration_steps.rb:132]: Expiry date doesn't need to be set. There is a step `I have an active registration` that could be used instead.

File: features/step_definitions/journey/registration_steps.rb [hunk 3]
```diff
@@ -147,34 +147,36 @@
...
 
-Given("I create an upper tier registration for my {string} business as {string}") do |business_type, account_email|
+Given("I create an upper tier registration for my {string} business") do |business_type|
   load_all_apps
   @tier = :upper
-  new_expiry_date = (DateTime.now + 30).to_s
-  seed_data = SeedData.new("#{business_type}_complete_active_registration.json", "accountEmail" => account_email, "contactEmail" => account_email, "expires_on" => new_expiry_date)
+  seed_data = SeedData.new("#{business_type}_complete_active_registration.json",
+                           "contactEmail" => Quke::Quke.config.custom["accounts"]["waste_carrier2"]["username"],
+                           "expires_on" => (DateTime.now + 30).to_s)
```
Comment (timstone) [inline features/step_definitions/journey/registration_steps.rb:155]: Setting expiry date isn't necessary

**Review by timstone — CHANGES_REQUESTED** [review]
Setting expires_on attribute for seed data isn't necessary as WCRS_REGISTRATION_RENEWAL_WINDOW is set to allow instant renewals.

File: features/step_definitions/journey/registration_steps.rb (hunk 2 above)
Comment (PaulDoyle-DEFRA) [reply features/step_definitions/journey/registration_steps.rb]: I've removed the unnecessary setting of expiry date throughout this file.

File: features/step_definitions/journey/registration_steps.rb [hunk 4]
```diff
@@ -150,9 +149,8 @@
 Given("I create an upper tier registration for my {string} business") do |business_type|
   load_all_apps
   @tier = :upper
-  seed_data = SeedData.new("#{business_type}_complete_active_registration.json",
-                           "contactEmail" => Quke::Quke.config.custom["accounts"]["waste_carrier2"]["username"],
-                           "expires_on" => (DateTime.now + 30).to_s)
+  new_expiry_date = (DateTime.now + 30).to_s
```
Comment (timstone) [inline features/step_definitions/journey/registration_steps.rb:152]: Did you intend on leaving this expiry date in?

**Review by timstone — APPROVED** [review]

**Pull request merged**